packages/next/src/utils.ts

```diff
@@ -2341,6 +2342,11 @@ export async function getMiddlewareBundle({
                   ...wasmFiles,
                   ...assetFiles,
                 },
+                regions:
+                  typeof edgeFunction.userConfig === 'object' &&
+                  'regions' in edgeFunction.userConfig
+                    ? (edgeFunction.userConfig.regions as any)
+                    : 'auto',
```

Comment:
Why not `edgeFunction.userConfig?.regions ?? 'auto'`? Do we need to accept `undefined` as a valid value?

Reply:
Maybe I was too strict here, I wanted to make sure that `edgeFunction.userConfig` is indeed an object, but you're right. I'll change <picture data-single-emoji=":relieved-telegram:" title=":relieved-telegram:"><img class="emoji" width="20" height="auto" src="https://single-emoji.vercel.app/api/emoji/eyJhbGciOiJkaXIiLCJlbmMiOiJBMjU2R0NNIn0..-S_kEm7Z9jgOzvu-.vA8-AIEc7Y3cmtLk4gMcDqkMLsLEUVzgKuJUzLz6j2G7BEoThlTEqqFSn70Yjgi4GO4V_x4MTcior2F81_pu8hV6PXUd3dWfLmhjnaBIwDEQJdlRcmyrcSeY-8snE4VwMMEG.JYrY2eQoi8cjHN2CrhmNPw" alt=":relieved-telegram:" align="absmiddle"></picture>